external/proto/omniledger.proto

```diff
@@ -122,42 +122,34 @@ message Proof {
 
 // Instruction holds only one of Spawn, Invoke, or Delete
 message Instruction {
-  // InstanceID holds the id of the existing object that can spawn new objects.
-  // It is composed of the Darc-ID + a random value generated by OmniLedger.
-  required InstanceID instanceid = 1;
-  // Nonce is monotonically increasing with regard to the darc in the instanceID
-  // and used to prevent replay attacks.
-  // The client has to track which is the current nonce of a darc-ID.
+  // InstanceID is either the object that can spawn a new object, or a the object
+  // that will be invoked or deleted.
+  required bytes instanceid = 1;
+  // Nonce is monotonically increasing with regard to the Darc controlling
```

Comment:
Is this used now? Or are we still ignoring it and hoping for a divine inspiration (I still don't have one ;)?

Reply:
Is what used? The nonce?

Reply:
Yes, the nonce... But as far as I saw in the code, it's not used yet in a way that makes it secure. Anyway - that's not the purpose of this PR.